question-servers/freeform.js

```diff
@@ -56,6 +57,13 @@ module.exports = {
             return '';
         } else if (phase == 'file') {
             return '';
+        } else if (phase == 'get_dependencies') {
```

Comment:
Yeah, Tim and I went back and forwards on this. For `file()` we decided to verb the noun rather than calling it `get_file()`, which means `dependencies()` would be consistent.